src/main/java/com/stripe/model/ChargeRefundCollectionDeserializer.java

```diff
@@ -18,6 +20,7 @@ public ChargeRefundCollection deserialize(JsonElement json, Type typeOfT, JsonDe
 		throws JsonParseException {
 		Gson gson = new GsonBuilder()
 			.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+			.registerTypeAdapter(ExpandableField.class, new ExpandableFieldDeserializer())
```

Comment:
Yeah, +1. The thing where we manually instantiate new GSONs all over the place that only have a set of hand-picked custom deserializer support seems super brittle to me and we should fix it, but this seems like the right thing to do for now.